components/ui/EditorButtons.tsx

```diff
@@ -30,6 +35,12 @@ const EditorButtons = ({ shiftRight }: EditorButtonsProps) => {
   function call<T>(command: CmdKey<T>, payload?: T) {
     return editor.get()?.action(callCommand(command, payload));
   }
+  // Function to insert image
+  const link: any = {
```

Comment:
```suggestion
  const link = {
```

You don't need to put `:any` here. Static variables automatically infer types from the assignment. So the type is actually:

```
{
url: string;
title: string;
}
```